idiss-csharp/IdissExample/IdissExample.csproj

```diff
@@ -6,7 +6,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This sort of change is not ideal without checking with the consumers first.

This idiss library was developed for DTS specifically first so we should make sure ideally that they can still use it.

If NET 5 is no longer supported I assume that is the case.

Reply:
Yes, that makes sense.
I have brought this to their attention and there should be no issue.